README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
+Use [Git for Windows](https://gitforwindows.org/) to prevent interference with the existing Windows environment and to have _Git Bash_ and _Git GUI_ launch commands available on the shortcut menu.
 
-## Build on Windows
+Open the Git Setup file downloaded from the Git for Windows site and use the following settings during the installation wizard:
 
-Windows does not provide the out-of-the-box Ruby, so we recommend to use a virtual environment with Docker or Vagrant described above.
-But if none of the ways work for you, consider using the following tools.
+-  select **Use Git from Git Bash only**
+-  select **Checkout as-is, commit Unix-style line endings**
+-  select your preferred editor (can use Nano, Notepad++, or VIM)
+-  select **Enable symbolic links**
 
-- [Git for Windows](https://git-scm.com/download)
-- [Chocolatey](https://chocolatey.org)
+Although you can install Git using Chocolatey, we chose to install _Git for Windows_ independently for more control of the installation settings.
 
-### Install Chocolatey
+#### Set up SSH key
 
-1. Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+1.  Open Git Bash. The **Git Bash** executable is on the shortcut menu.
 
-2. [Install Chocolatey](https://chocolatey.org/install).
+1.  Create a working directory for your Git repositories and change to the new directory.
 
-3. Verify Chocolatey was added to the environment variables. In the Windows CMD console, type `echo %path%`.
-   You should see `C:\ProgramData\chocolatey\bin` in the path.
+    ```bash
+    mkdir <directory-name>
+    ```
 
-4. Close and reopen the command prompt before using `choco` commands.
+1.  Follow the [Generating a new SSH](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) instructions.
 
-### Install Ruby package
+### Clone and build the DevDocs repository
 
-1. Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+You may have to close and reopen the Git Bash application after the Choco installations.
 
-1. Install the ruby package:
+1.  Open Git Bash. The **Git Bash** executable is on the shortcut menu.
 
-   ```shell
-   > choco install ruby --version 2.5.3.101
-   ```
+1.  Change to the directory you created for Git repositories and clone the DevDocs repository.
 
-    Reopen the Command prompt to enable the installation.
+    ```bash
+    git clone git@github.com:magento/devdocs.git
+    ```
 
-> **NOTE**  
-> If you encounter problems with Ruby, or the `gem` command is not recognized, you can install the `rubyinstaller-devkit.exe` development kit located in the `c:\ProgramData\chocolatey\bin` folder.
+1.  Change to the `devdocs/` directory.
 
-1. Install the Ruby development kit
+1.  Install [Bundler](https://bundler.io).
 
-   ```shell
-   ridk install
-   1 - MSYS2 base installation
-   2 - MSYS2 system update (optional)
-   3 - MSYS2 and MINGW development toolchain
+    ```bash
+    gem install bundle
```

Comment:
`gem install bundler` will install the latest Bundler which is of major version 2.
There is a known issue with RubyGems versions older than 3.